README.md

```diff
@@ -30,6 +30,65 @@ To start using `mender-cli`, we recommend that you begin with the
 You can find the latest `mender-cli` binaries in the [Downloads page on Mender
 Docs](https://docs.mender.io/downloads).
 
+## Enabling Bash auto-complete
+
+The `mender-cli` tool can be enabled to support shell autocompletion, like you
+are used to for your regular tools, like `git`, `cd`, etc. In order to enable
+this functionality the `mender-cli` tool has to be run with the
+`MENDER_CLI_GEN_AUTOCOMPLETE` environment variable set to a value. Example:
+
+```console
+MENDER_CLI_GEN_AUTOCOMPLETE mender-cli -h
```

Comment:
```suggestion
MENDER_CLI_GEN_AUTOCOMPLETE=something mender-cli -h
```

Reply:
Lol, yeh. Sry. 
Adding the `MENDER_CLI_GEN_AUTOCOMPLETE=true`.
Thanks